app/actions.py

```diff
@@ -629,10 +629,40 @@ def cursorSelectPageUp(self):
     Returns:
       None.
     """
-    if self.selectionMode == app.selectable.kSelectionNone:
-      self.doSelectionMode(app.selectable.kSelectionCharacter)
+    if self.selectionMode != app.selectable.kSelectionCharacter:
+      self.selectionCharacter()
+    self.__cursorPageUp()
+
+  def cursorAltSelectPageDown(self):
+    """
+    DESCRIPTION HERE
+
+    Args:
+      None.
+
+    Returns:
+      None.
+    """
+    if self.selectionMode != app.selectable.kSelectionBlock:
+      self.selectionBlock()
+    self.__cursorPageDown()
+
+  def cursorAltSelectPageUp(self):
```

Comment:
how about the name cursorSelectBlockPageUp

Reply:
Sounds good to me.